user-portal/patternfly4/create-new-vm-pf4.md

```diff
@@ -0,0 +1,12 @@
+### Create New VM
+The PatternFly 4 version of the create new VM modal features the same functionality as the current one but an updated look. 
```

Comment:
Something to consider is if they go for this update maybe they will want to align with the CNV in page wizard design. Not for this PR but a note for later :)

Reply:
We can definitely take a second look at it so it aligns more with CNV in the future. A lot of this PF4 work is speculative.